streamz/sources.py

```diff
@@ -503,7 +507,7 @@ def checkpoint_emit(_part):
                     try:
                         low, high = self.consumer.get_watermark_offsets(
                             tp, timeout=0.1)
-                    except (RuntimeError, ck.KafkaException):
+                    except (RuntimeError, ck.KafkaException, ValueError):
                         continue
```

Comment:
Do we need a sleep statement here to prevent a spam query for offsets?  Also, I wonder if we need logging to here to explain the reason for the `ValueError`